assets/airports/eidw.json

```diff
@@ -883,124 +1076,361 @@
             "end":    [["N53d26m13.16", "W6d15m43.12", "217ft"], ["N53d25m11.66", "W6d14m58.54", "202ft"]],
             "length":      2.072,
             "ils":         [true, true]
+        },
+        {
+            "name":        ["10X", "28X"],
```

Comment:
Instead of these runways, couldn't we just have the spawn patterns only spawn the right type of aircraft on the appropriate departure procedures? Yes, I know this would mean adding two new fleets in each of the used airlines, but it seems like a more logical and proper solution.